AutomatedTesting/Gem/PythonTests/scripting/TestSuite_Periodic.py

```diff
@@ -412,6 +422,7 @@ def test_ScriptCanvas_TwoComponents_InteractSuccessfully(self, request, workspac
             auto_test_mode=False,
             timeout=60,
         )
+    """
```

Comment:
Let's add a comment here as to why this is blocked out with a leading GHI using the `o3de/oded#1324` pattern

Reply:
This was actually left in by mistake. This test runs fine from multitest framework. I just didn't erase it from hydra framework while testing in case I needed to make comparisons.